<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverListButtonWidget</class>
 <widget class="QWidget" name="OverListButtonWidget">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="btnAdd">
     <property name="icon">
      <iconset resource="bibolamazi_res.qrc">
       <normaloff>:/pic/lstbtnadd.png</normaloff>:/pic/lstbtnadd.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnEdit">
     <property name="icon">
      <iconset resource="bibolamazi_res.qrc">
       <normaloff>:/pic/lstbtnedit.png</normaloff>:/pic/lstbtnedit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnRemove">
     <property name="icon">
      <iconset resource="bibolamazi_res.qrc">
       <normaloff>:/pic/lstbtnremove.png</normaloff>:/pic/lstbtnremove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="bibolamazi_res.qrc"/>
 </resources>
 <connections/>
</ui>
